docs/userguide.rst

```diff
@@ -0,0 +1,204 @@
+.. _userguide:
+
+User guide
+==========
+
+Prerequisites
+-------------
+
+Currently reana-cluster command line tool supports deploying
+REANA components only to `Kubernetes <https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/>`_
+infrastructure, so in order to deploy REANA components one needs to have a
+working Kubernetes deployment.
+
+reana-cluster utilizes configuration files of `kubectl -tool <https://kubernetes.io/docs/user-guide/kubectl-overview/>`_
+to set up a connection to Kubernetes infrastructure, so in order
+to use reana-cluster one needs to have kubectl installed
```

Comment:
Well `kubectl` itself is not the requirement, just configuration files in kubeconfig format. `kubectl` configuration file is located by default in `$HOME/.kube/config, which python-kubernetes client will try to pickup automatically in case we don't specify it with `reana-cluster.yaml`.

Reply:
I don't know if Minikube check that `kubectl` is available or does it just modify `$HOME/.kube/config` and add authentication certificates under `$HOME/.kube`